libcudacxx/include/cuda/std/__mdspan/extents.h

```diff
@@ -140,6 +140,7 @@ class extents
   // internal typedefs which for technical reasons are public
   using __storage_t =
     __detail::__partially_static_sizes_tagged<__detail::__extents_tag, index_type, size_t, _Extents...>;
+  using __indices_t = _CUDA_VSTD::integer_sequence<size_t, _Extents...>;
```

Comment:
Could you make this a template and then have `__indices_t<_Extents...>{}` and `__indices_t<_OtherExtents...>{}`

Reply:
no, the point of this change is because msvc gets confused by the pack expansion at the point of use.

Reply:
Oh yeah i love that